### ISSUE TITLE
Pretrained Model

### ISSUE THREAD

**Issue by dleam** [lyndonzheng/Synthetic2Realistic#13]
@lyndonzheng Excellent Job! Before I read your paper, I just have the same idea with you to transform the synthetic image to the real one and use the transformed image for the depth estimation task. Anyway, I have a question about you code. How do you choose the real dataset for training the Transform Gan Net. I have tried to use all the KITTI raw dataset and the all the synthetic dataset (with normal weather). The results seem to be not good after several epochs' training. Can you tell me how to choose the dataset to reproduce your results in the paper or provide a pretrained model. I'm really interesting in your work and want to do some further research on it. I think your proposed unsupervised method for depth estimation.

**Comment by lyndonzheng** [comment]
@dleam Thanks for your interesting. Since the original code  of the paper is a mess, I rewrite the code. And I was busy with other tasks, I just run the new code for the indoor scene. In the original paper, the synethetic dataset is raw vKITTI dataset. I have finished the new task, so I will retrain the model and release them soon.

**Comment by lyndonzheng** [comment]
@dleam I have upload the pretrained model for Outdoor KITTI dataset of our full T2Net. The depth estimation has a very tiny difference with the original paper, you can check it. The translated results are reasonable for the normal weather of vKITTI.  I train the model with fixed learning rate for first 10 epoches and a linearly decaying rate for the next 10 epoches. The training parameter has a little different with the original paper, that's because we rewrite partial code for the easily read. If you still has questions about the code, please feel free to send information to me. We can discuss more.
